solutions/standard/variables.tf

```diff
@@ -237,3 +237,9 @@ variable "enable_elser_model" {
   description = "Set it to true to install and start the Elastic's Natural Language Processing model. [Learn more](https://cloud.ibm.com/docs/databases-for-elasticsearch?topic=databases-for-elasticsearch-elser-embeddings-elasticsearch)"
   default     = false
 }
+
+variable "elser_model_type" {
+  type        = string
+  description = "Trained ELSER model to be used for Elastic's Natural Language Processing. [Learn more](https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html)"
+  default     = "elser_model_2_linux-x86_64"
```

Comment:
add validation for the supported types, and include possible values in the var descrption